Gemfile.lock

```diff
@@ -95,9 +95,9 @@ GEM
     grape-entity (0.5.1)
       activesupport
       multi_json (>= 1.3.2)
-    grape-swagger (0.23.0)
+    grape-swagger (0.25.3)
```

Comment:
Hi, this is looking great, but would you mind removing the `Gemfile.lock` from your PR, as I think it's just a version tweak? https://stackoverflow.com/questions/215718/reset-or-revert-a-specific-file-to-a-specific-revision-using-git

Thanks a lot!!